### ISSUE TITLE
Publish to the Github Registry

### ISSUE THREAD

**Issue by dr3** [BBC-archive/psammead#538]
Blocked on access to the github registry

**Is your feature request related to a problem? Please describe.**
It is likely that we might get Github registry access soon, would be nice to publish to it (as well as npm)

**Describe the solution you'd like**
Update our publish script to work with
```
"publish": "npm run build && npm run publish:npm && npm run publish:github",
"publish:npm": "node scripts/publish --registry=https://registry.npmjs.org",
"publish:github": "node scripts/publish --registry=https://npm.pkg.github.com",
```
or some variation of that

**Testing notes**
[Tester to complete]

Dev insight: Unit test and manual testing

**Additional context**
We should confirm with @benjaminhobbs before doing this

**Comment by benjaminhobbs** [comment]
Perfectly fine. Is there a **really low priority** label? :)

**Comment by jamesbhobbs** [comment]
@joshcoventry this can probably be closed ;)

**Comment by joshcoventry** [comment]
@benjaminhobbs I agree, thanks!
